jupyter/jsl_installation_script_ubuntu.sh

```diff
@@ -88,6 +92,12 @@ install_ocr(){
 
 }
 
+fetch_license_from_jsl(){
```

Comment:
It won't be easy to support. Can we use heredoc syntax here?

Reply:
`fetch_license_from_myjsl` would be better, I think.

Reply:
What about heredoc?

Reply:
Do we need heredoc (<<)?